documents/api/filtersets.py

```diff
@@ -5,6 +5,9 @@
 
 class DocumentFilterSet(filters.FilterSet):
     user_id = filters.UUIDFilter(field_name="user__uuid")
+    service_name = filters.CharFilter(
+        field_name="service__name", label="Service's name"
```

Comment:
Do you want this to be exact filter?

Reply:
I think this is a subject for change, but for now I think it should be exact name of the service.